addon/components/lazy-options/template.hbs

```diff
@@ -0,0 +1,43 @@
+<!--  https://github.com/cibernox/ember-power-select/blob/master/addon/templates/components/power-select/options.hbs -->
+{{#if select.loading}}
```

Comment:
A couple of things/questions:

1. Did you mean to have two different loading messages here? 
2. Is there a difference between `select.loading` and `isLoading`?  If they're both the same, it might make more sense to use one or the other, or just use `isLoading` as an alias of `select.loading`.

Reply:
This template is mostly the same as the options template for ember-power-select (https://github.com/cibernox/ember-power-select/blob/master/addon/templates/components/power-select/options.hbs). `select.loading` is used by ember-power-select to know whether the options are loading. `isLoading` is for whether additional options is currently loading.

Reply:
In that case, would it work to make a `computed.or` in `lazy-options/component.js` that looked at `isLoading` and `select.loading`?  Then you can just use that property to determine if anything is loading at all.  

That or you could maybe only have one statement in the template that looked at `{{#if (or select.loading isLoading)}}`

Reply:
The idea is that they are indeed two different loading messages for different circumstances.